content/src/certifications/dbe.md

```diff
@@ -36,6 +36,8 @@ A DBE must meet all of the following:
 - Requires submission of documents to support personal net worth (PNW) and DBE statutory cap
 - Applicant must participate in the on-site interview proceedings
 
-> **After you complete this, you will have:**
->
-> - A Disadvantaged Business Enterprise certificate
+:::callout{ showHeader="true" headerText="" showIcon="true" calloutType="conditional" }
```

Comment:
pebble - header text was not correctly added

Reply:
I thought we were only replacing the following text with the default? `Once completed, you will have:`

Reply:
it's also not clear why show icon is true in some instances, and false in others

Reply:
JP's advice, synced with the content team, is that anything that had close enough wording would just default to the default heading text